### PR TITLE
asyncsnmp depends on sonic-utilities so it is possible to import sonic_psu

### PR DESCRIPTION
**Pull request by qiluo-msft**
**- What I did**

**- How I did it**

**- How to verify it**
Build snmp docker and test importing in python3.6
**- Description for the changelog**

**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by qiluo-msft** [comment]
Waiting sonic-slave building and will retest this

**Comment by qiluo-msft** [comment]
Retest this please

**Comment by qiluo-msft** [comment]
Retest this please

File: slave.mk [hunk 1]
```diff
@@ -306,7 +306,14 @@ $(addprefix $(PYTHON_WHEELS_PATH)/, $(SONIC_PYTHON_WHEELS)) : $(PYTHON_WHEELS_PA
...
 	if [ -f ../$(notdir $($*_SRC_PATH)).patch/series ]; then QUILT_PATCHES=../$(notdir $($*_SRC_PATH)).patch quilt push -a; fi
-	python$($*_PYTHON_VERSION) setup.py test $(LOG)
+	# TRICK! TRICK! TRICK!
+	# Why we test twice?
+	#  The unittest module from 3.4.2 will not treat setup.py - setup - test_suite correctly
+	#  It insists on load all the unit test on the first run.
+	# How to remove the ugly second test?
+	#  1. The unittest module from 3.6 does well
+	#  2. Not to use conditional test_suite, so recursively refine all the dependencies
+	python$($*_PYTHON_VERSION) setup.py test $(LOG) || python$($*_PYTHON_VERSION) setup.py test $(LOG)
```
Comment (lguohan) [inline slave.mk]: I would add a build option to not to test in this case.

File: rules/sonic-utilities.mk [hunk 2]
```diff
@@ -4,3 +4,10 @@ SONIC_UTILS = python-sonic-utilities_1.1-1_all.deb
 $(SONIC_UTILS)_SRC_PATH = $(SRC_PATH)/sonic-utilities
 $(SONIC_UTILS)_WHEEL_DEPENDS = $(SONIC_CONFIG_ENGINE)
 SONIC_PYTHON_STDEB_DEBS += $(SONIC_UTILS)
+
+# Build sonic-utilities into python3 wheel, so we can use PSU code
+# Note: _DEPENDS macro is not defined
+SONIC_UTILS_PY3 = sonic_utilities-1.1-py3-none-any.whl
+$(SONIC_UTILS_PY3)_SRC_PATH = $(SRC_PATH)/sonic-utilities
+$(SONIC_UTILS_PY3)_PYTHON_VERSION = 3
```
Comment (lguohan) [inline rules/sonic-utilities.mk]: We can add 
$(SONIC_UTILS_PY3)_TEST = n to disable the test.

It is more useful.

File: slave.mk (hunk 1 above)
Comment (lguohan) [inline slave.mk]: $(if $($*_TEST), python$($*_PYTHON_VERSION) setup.py test $(LOG))

Comment (lguohan) [reply slave.mk]: https://www.gnu.org/software/make/manual/html_node/Conditional-Functions.html#Conditional-Functions

**Comment by stcheng** [comment]
wait for build

**Pull request merged**